app/models/assignment.rb

```diff
@@ -44,6 +44,7 @@ def persist_submission!(submission)
     transaction do
       messages.destroy_all if submission_id.present?
       update! submission_id: submission.id
+      update! submitted_at: DateTime.now
```

Comment:
Perhaps we should use `DateTime.current`

See https://stackoverflow.com/questions/12748626/when-should-datetime-now-utc-vs-time-current-utc-be-used-in-rails?answertab=votes#tab-top

Reply:
Mmm, interesting, I didn't know. Nevertheless, we are using `DateTime.now` everywhere, so I'll open a separate issue for that one!